bundlegen/src/main/java/com/citrix/microapps/bundlegen/bundles/BundlesLoader.java

```diff
@@ -362,6 +366,10 @@ private static boolean isTranslationValid(TemplateFile template, ModelTranslatio
         return Optional.empty();
     }
 
+    private static String replaceWhitespacesWithUnderscores(String text) {
+        return text.replaceAll(" ", WORD_SEPARATOR);
```

Comment:
a little robustness to avoid NPE if it's not overengineering:
```
if (StringUtils.isNotBlank(text)) {
  return text.trim().replaceAll("\\s+", WORD_SEPARATOR);
} else {
  return text;
}
```

Reply:
Sounds reasonably, thank you. Potential NPE fixed.